config/environments/test.rb

```diff
@@ -49,4 +49,8 @@
 
   # Use separate cache stores for parallel_tests
   config.cache_store = :file_store, Rails.root.join("tmp", "cache", "paralleltests#{ENV['TEST_ENV_NUMBER']}")
+
+  # Disables automatic creation of the test DB so that it has to be explicitly
+  # generated before running tests.
+  config.active_record.maintain_test_schema = false
```

Comment:
When I tested yesterday and commented:

> >  rake test still works OK though?
> Oh no, it doesn't.

It seems that this also requires anyone using plain old `rake test` now needs to prepare the test DB manually.  They'd probably need to do it every time there's a migration too?